src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs

```diff
@@ -96,9 +96,8 @@ public AdvancedOptionPageControl(OptionStore optionStore, IComponentModel compon
             // Quick Actions
             BindToOption(ComputeQuickActionsAsynchronouslyExperimental, SuggestionsOptions.Asynchronous, () =>
```

Comment:
in main and main-vs-deps we want to remove the entire BindToOption, and these shouldn't be in the xaml anymore either.